src/source/Metrics/Metrics.c

```diff
@@ -140,6 +140,26 @@ STATUS getRtpInboundStats(PRtcPeerConnection pRtcPeerConnection, PRtcRtpTranscei
     return retStatus;
 }
 
+STATUS getDataChannelStats(PRtcPeerConnection pRtcPeerConnection, PRtcDataChannelStats pRtcDataChannelStats)
+{
+    STATUS retStatus = STATUS_SUCCESS;
+    PKvsDataChannel pKvsDataChannel = NULL;
+    UINT32 numberOfDataChannels = 0;
+    PKvsPeerConnection pKvsPeerConnection = (PKvsPeerConnection) pRtcPeerConnection;
+    CHK(pRtcPeerConnection != NULL || pRtcDataChannelStats != NULL, STATUS_NULL_ARG);
```

Comment:
So if pRtcPeerConnection is null then we should ensure pRtcDataChannelStats is NOT? I think you meant &&

Reply:
You are right. I did mean &&.